docs/api-reference/next.config.js/eslint-warnings-errors.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Learn to opt-in and out of ESLint during development mode and production builds.
+---
+
+# ESLint Warnings and Errors
+
+## During builds
+
+Next.js fails your **production build** (`next build`) when ESLint errors are present in your
+project.
+
+If you'd like Next.js to dangerously produce production code even when your application has errors,
+you can disable ESLint running during the build process.
+
+> Be sure you are running ESLint as part of your build or deploy process, otherwise this can be
```

Comment:
IMO, this repeats the line before. Maybe we change this to say _why_ it's dangerous? 

> It's recommended to run ESLint as part of the production build process to ensure your application is resilient against runtime issues.

Or similar.

Reply:
Updated, thanks!